src/locales/en/translation.json

```diff
@@ -141,6 +143,7 @@
     "tokenInstance": "TokenID [<InstanceLink />] <TickerLink/>",
     "tokens": "Tokens",
     "totalVotes": "Total votes",
+    "recentTransactions": "Recent transactions",
```

Comment:
> One thought I had was if it would it be clearer to rename this row from "Transaction Action" to "Recent transactions"?

@donouwens these aren't transactions; the whole page is one transaction. Elsewhere we named this subset of events "Transfers". If we intend to show more than just transfers here then we need some other word :shrug: 

And I think "Recent" is misleading too

Reply:
Maybe "Summary"